### DISCUSSION TITLE
MyBB Convertor

### DISCUSSION THREAD

**Discussion by GershonR** [NamelessMC/Nameless#2122] [Ideas]
### Template:
* What kind of issue is it (Question, Bug, etc.)? Suggestion
* In which version of NamelessMC did it occur (1.x.x)? 1.0.15
* How would you describe the issue? 

Currently using a MyBB forum, I have deleted the forum folder for Nameless for now so I can use my own forum, I would like to re-integrate that forum with NamelessMC

**Comment by BaxAndrei** [comment]
I use mybb, it would be a great idea.
